corehq/apps/export/export.py

```diff
@@ -456,11 +456,11 @@ def _get_base_query(export_instance):
 
 
 @datadog_track_errors('rebuild_export')
-def rebuild_export(export_instance, filters, progress_tracker):
+def rebuild_export(export_instance, progress_tracker):
     """
     Rebuild the given daily saved ExportInstance
     """
-    filters = filters or export_instance.get_filters()
+    filters = export_instance.get_filters()
```

Comment:
nice cleanup in this commit. it's a bit hard to wrap my head around this, but do you know why this logic used to be here and why filters were previously passed in separately?

Reply:
I think maybe the answer is that filters was never actually passed in and the reason must be "some legacy thing that no longer matters"

Reply:
> "some legacy thing that no longer matters"

That describes a lot of this code.  I suspect that at one point it was passed in (prior to [this commit](https://github.com/dimagi/commcare-hq/commit/54ca07a4c5c24a6c7935d4f2bf7e630d8f33802b) perhaps), then that was probably phased out, but the optional arg was never removed.